Uchu.World/Handlers/Commands/OperatorCommandHandler.cs

```diff
@@ -236,5 +237,57 @@ public string Target(string[] arguments, Player player)
 
             return $"Target: {baseCombatAiComponent.Target}";
         }
+
+
+        [CommandHandler(Signature = "execute", Help = "Execute a command as another player", GameMasterLevel = GameMasterLevel.Admin)]
+        public async Task<string> Execute(string[] arguments, Player executor)
+        {
+            List<Player> players = new List<Player>();
+            if (arguments[0] == "*") {
+                foreach (var player in executor.Zone.Players) {
+                    players.Add(player);
+                }
+            } else {
+                var player = executor.Zone.Players.FirstOrDefault(p => p.Name == arguments[0]);  // This throws an error if no character is found :shrug:
+                if (player == default) return $"No player named {arguments[0]}";
+                players.Add(player);
+            }
+
+            await using var ctx = new UchuContext();
+            var character = await ctx.Characters.Include(c => c.User).FirstAsync(
+                c => c.Id == executor.Id
+            );
+
+            var _com = new List<string>(arguments);
```

Comment:
Bit of a odd variable name, if you meant command I would just use `command` here